src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -21,13 +21,25 @@ private static List<Point> makePoints(int width, BooleanGenerator booleanGenerat
         .collect(ArrayList::new, (list, element) -> {
           if (list.isEmpty() || list.get(list.size() - 1).isNotBridge()) {
             list.add(Point.of(element));
-          } else {
-            list.add(Point.of(FALSE));
+            return ;
           }
+          list.add(Point.of(FALSE));
         }, ArrayList::addAll);
   }
 
   public List<Point> getPoints() {
     return points;
   }
+
+  public int move(int index) {
+    if(index > 0 && !points.get(index-1).isNotBridge() ) {
```

Comment:
해당 라인만 보고선 이게 어떤 경우인지 사람의 눈으로는 쉽게 감이 오지 않을 것 같아요.
메서드 추출을 통해 이름을 부여하여 가독성을 높여보면 어떨까요?
```suggestion
    if (canMoveLeft(index)) {
```